resources/social-media.md

```diff
@@ -0,0 +1,29 @@
+# RustBridge Social Media Guide
+Social Media can be a very powerful tool to leverage your event, but also to motivate attendees to share their results! Because it’s so powerful, it’s very helpful and sometimes necessary to set concrete rules for usage by not only organizers and teachers, but also attendees.
+
+
+## Photo Guidelines
+
+This is the part that is necessary. You want to make sure that your event is governed by a photo taking policy, meaning that you want to ask for people’s *consent* before you take their picture. The same thing goes for attendees taking pictures.
+
+Assume that every individual person is **not** ok with having their picture taken until you ask them to. The easiest way to do this is to sort this out at the beginning by asking the crowd of attendees if they’re okay with it. If not everyone is, *don’t* call them out — tell them that it’s okay, and ask the rest of the group to either be careful with or refrain from taking pictures with people in them altogether.
```

Comment:
sometimes people are not happy saying in front of the group that they are uncomfotable, providing a backchannel like a slack can be very useful to allow people to do this. it also may be worth asking in the emails ahead of time so students can share their preference in private.